api/src/main/java/org/apache/commons/rdf/api/RDFSyntax.java

```diff
@@ -34,7 +152,7 @@
  *      1.1 Primer</a>
  * @see org.apache.commons.rdf.experimental.RDFParser
  */
-public enum RDFSyntax {
+  public enum OfficialRDFSyntax implements RDFSyntax {
```

Comment:
I also wanted to list the `IRI` identifying the syntaxes, but then I would need a complete `IRIImpl` within `api/` .. or move `W3CRDFSyntax` to `simple/`.  (or.. awkwardly.. ask for an `RDF` instance :))